sycl/test-e2e/AddressSanitizer/bad-context/bad-context-1.cpp

```diff
@@ -0,0 +1,16 @@
+// UNSUPPORTED: true
```

Comment:
Is this test pass on local machine? Is there plan to support this test?

Reply:
In fact, I don't know how to enable a lit test that needs "opencl:cpu" and "level:gpu" to be both present.

Reply:
I think you can add multiple `REQUIRES:` lines.

Reply:
Thanks, I'll try it

Reply:
> I think you can add multiple `REQUIRES:` lines.

That won't work - each `REQUIRES` is processed on a per-device basis, they won't be set simultaneously.

Reply:
Any other suggestion for this particular scenario? Is this possible using a single line? Aren't `cpu` and `gpu` excluding each other? And same for `opencl` and `level0`?

Reply:
We have this: https://github.com/intel/llvm/blob/cbffacd451b1cf695fc886029feedcbf11ec11e2/sycl/test-e2e/lit.cfg.py#L582-L586, maybe it can help.